code/__HELPERS/global_lists.dm

```diff
@@ -45,6 +45,12 @@
 		var/datum/scream_type/S = new spath()
 		GLOB.scream_types[S.name] = spath
 	sortList(GLOB.scream_types, /proc/cmp_typepaths_asc)
+
+	//Laugh types
+	for(var/spath in subtypesof(/datum/laugh_type))
```

Comment:
You're doing this the correct way! <3

Reply:
I'll be entirely honest with you here chief, most of this was basically me just adapting what Gandalf did for the screams, but yeah, I'm pretty happy with the results.